src/kotlin_editor_export_plugin.cpp

```diff
@@ -179,6 +177,17 @@ void KotlinEditorExportPlugin::_generate_export_configuration_file(jni::JvmType
         get_export_preset()->set_exclude_filter(get_export_preset()->get_exclude_filter() + "," + JVM_CONFIGURATION_PATH);
     }
 
+    if (const String build_dir = String {BUILD_DIRECTORY}.path_join("*"); !get_export_preset()->get_exclude_filter().contains(build_dir)) {
```

Comment:
You are inside the function that generate the configuration file here, so I don't think it's the best place.